sphinx/source/docs/user_guide/server.rst

```diff
@@ -687,19 +688,27 @@ include any or all of the following conventionally named functions:
 .. code-block:: python
 
     def on_server_loaded(server_context):
-        '''If present, this function executes when the server starts.'''
+        '''
+        If present, this function executes when the server starts.
+        '''
```

Comment:
FWIW we don't have docstrings formatted like this anywhere in the codebase, we always start with a comment on the same line as the `'''`

Reply:
I was hesitant about changing these this way, but breaking the latter two comments mid-sentence to respect the 80-char restriction felt 'ugly'. Especially compared to how neat they looked originally. That said, I can change them to be more in line with the rest of the docs.

Reply:
Turns out switching to single line just about does it in terms of character limit and is more conistent with other comments in the docs.